### PR TITLE
Spelling And Grammar Fixes

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes spelling, grammar, spacing.

## Why It's Good For The Game
Better user experience.

## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

**Comment by EvilJackCarver** [comment]
Don't be a tit. @waterwater1212 
@klorpa / @drexample I'm on holiday, so it'll be a few days before I review the changes.

**Comment by EvilJackCarver** [comment]
I'm back off holiday. Give me a few days to thumb through the changes, there's a lotta files changed here.

File: code/_helpers/names.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@ var/religion_name = null
...
+			name = pick("", "Stanford", "Dorf", "Alium", "Prefix", "Clowning", "Aegis", "Ishimura", "Scaredy", "Death-World", "Mime", "Honk", "Rogue", "MacRagge", "Ultrameens", "Safety", "Paranoia", "Explosive", "Neckbear", "Donk", "Muppet", "North", "West", "East", "South", "Slant-ways", "Widdershins", "Rimward", "Expensive", "Procreatory", "Imperial", "Unidentified", "Immoral", "Carp", "Ork", "Pete", "Control", "Nettle", "Aspie", "Class", "Crab", "Fist","Corrugated","Skeleton","Race", "Fatguy", "Gentleman", "Capitalist", "Communist", "Bear", "Beard", "Derp", "Space", "Spess", "Star", "Moon", "System", "Mining", "Neckbeard", "Research", "Supply", "Military", "Orbital", "Battle", "Science", "Asteroid", "Home", "Production", "Transport", "Delivery", "Extraplanetary", "Orbital", "Correctional", "Robot", "Hats", "Pizza")
```
Comment (EvilJackCarver) [inline code/_helpers/names.dm]: Where's the change on this line? GitHub isn't highlighting it.

File: code/game/objects/items/weapons/grenades/projectile.dm [hunk 2]
```diff
@@ -45,7 +45,7 @@
 	projectile_types = list(/obj/item/projectile/energy/flash)
 
 /obj/item/weapon/grenade/shooter/energy/tesla
-	name = "tesla grenade"
+	name = "Tesla grenade"
```
Comment (EvilJackCarver) [inline code/game/objects/items/weapons/grenades/projectile.dm]: Can we get an `\improper` here for the examine text?

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -27,8 +27,8 @@
 	item_state_slots = list(slot_r_hand_str = "scratch", slot_l_hand_str = "scratch")
 
 /obj/item/clothing/under/sl_suit
-	desc = "It's a very amish looking suit."
-	name = "amish suit"
+	desc = "It's a very Amish looking suit."
+	name = "Amish suit"
```
Comment (EvilJackCarver) [inline code/modules/clothing/under/miscellaneous.dm]: `\improper` here as well, for the examine text?

File: code/modules/integrated_electronics/passive/power.dm [hunk 4]
```diff
@@ -94,7 +94,7 @@
 
 // For fat machines that need fat power, like drones.
 /obj/item/integrated_circuit/passive/power/relay
-	name = "tesla power relay"
+	name = "Tesla power relay"
```
Comment (EvilJackCarver) [inline code/modules/integrated_electronics/passive/power.dm]: `\improper` here as well?

File: code/modules/lore_codex/news_data/main.dm [hunk 5]
```diff
@@ -137,7 +137,7 @@
...
+	data = "The recent détente with the Almach Association has prompted easier communications with Rim governments. Loved ones separated by the cordon have a chance to communicate once more, trade is posed to recommence, and light has been shed on the conditions of Shelf, Relan, and Angessa's Pearl. Amid this light is a patch of darkness. The fourth major polity of the Association, the Aetolian Council remains inscrutable, with no publicly-available information in the system available after their purge of corporate loyalists during the Gray Hour. What reports do exist are rumors within the Rim of Aetolian projects to create a new, hardier strain of Promethean, potentially one in flagrant violation of the Five Points of Human Sanctity. It is also known that Aetolus is a contributor to the personnel of the military vessels that even now are active in the Golden Crescent, although no so-called \"Aeto-Prometheans\" are believed to be active outside of the rim at this time.\
```
Comment (EvilJackCarver) [inline code/modules/lore_codex/news_data/main.dm]: Where's the change in this line?

File: code/modules/mining/fulton.dm [hunk 6]
```diff
@@ -1,7 +1,7 @@
 var/global/list/total_extraction_beacons = list()
 
 /obj/item/extraction_pack
-	name = "fulton extraction pack"
+	name = "Fulton extraction pack"
```
Comment (EvilJackCarver) [inline code/modules/mining/fulton.dm]: `\improper` here?

File: code/modules/mining/fulton.dm [hunk 7]
```diff
@@ -159,8 +159,8 @@ var/global/list/total_extraction_beacons = list()
 		qdel(src)
 
 /obj/structure/extraction_point
-	name = "fulton recovery beacon"
-	desc = "A beacon for the fulton recovery system. Activate a pack in your hand to link it to a beacon."
+	name = "Fulton recovery beacon"
```
Comment (EvilJackCarver) [inline code/modules/mining/fulton.dm]: `\improper`

File: code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm [hunk 8]
```diff
@@ -306,16 +306,16 @@
...
-/obj/item/weapon/gun/energy/taser/mounted/cyborg/ertgun //Not a taser, but it's being used as a base so it takes energy and actually works.
+/obj/item/weapon/gun/energy/taser/mounted/cyborg/ertgun //Not a Taser, but it's being used as a base so it takes energy and actually works.
 	name = "disabler"
 	desc = "A small and nonlethal gun produced by NT.."
 	icon = 'icons/mob/dogborg_vr.dmi'
 	icon_state = "projgun"
 	fire_sound = 'sound/weapons/eLuger.ogg'
 	projectile_type = /obj/item/projectile/beam/disable
-	charge_cost = 240 //Normal cost of a taser. It used to be 1000, but after some testing it was found that it would sap a borg's battery to quick
+	charge_cost = 240 //Normal cost of a Taser. It used to be 1000, but after some testing it was found that it would sap a borg's battery to quick
```
Comment (EvilJackCarver) [inline code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm]: >"... sap a borg's battery to quick"

'Too'?

File: code/modules/lore_codex/news_data/main.dm (hunk 5 above)
Comment (klorpa) [reply code/modules/lore_codex/news_data/main.dm]: detente -> détente

File: code/_helpers/names.dm (hunk 1 above)
Comment (klorpa) [reply code/_helpers/names.dm]: Corrogated -> Corrugated

File: code/modules/mob/living/carbon/human/life.dm [hunk 9]
```diff
@@ -219,7 +219,7 @@ var/last_message = 0
 		if(10 <= rn && rn <= 12)
 			if(prob(50))
 				src << "<span class='danger'>You suddenly black out!</span>"
-				Paralyse(10)
+				Paralyze(10)
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/life.dm]: The proc is `Paralyse()`, changing this without changing the proc name is what's breaking it

**Comment by EvilJackCarver** [comment]
I've completely forgotten about this PR being open, a lot of shit's happened in real life that's demanded my attention and in the time it's taking to deal with that, I haven't really touched code up until about a week or so ago.

I should probably make a note that this branch is currently under a code freeze (and is also not really in active use at the moment, though I expect that at some time Nestor might want to pick it back up), so the stuff going on here isn't going to have much effect until that happens.

I thumbed through the Travis log and highlighted one of two changes in the same file (both changes are the same, Paralyse → Paralyze) that're keeping Travis from passing. As soon as I notice that they pass, I'll merge it in. (If I don't within a day or so, yell at me!)

**Pull request merged**